cpp/include/nvstrings/NVText.h

```diff
@@ -82,6 +82,16 @@ class NVText
      */
     static unsigned int token_count( NVStrings& strs, const char* delimiter, unsigned int* results, bool devmem=true );
 
+     /**
+     * @brief Computes the number of tokens in each string.
+     * @param strs Strings to tokenize.
+     * @param delimiter NVStrings object containing String or character used to identify tokens.
+     * @param[in,out] results Array of counts, one per string.
+     * @param devmem True if results in device memory.
+     * @return 0 if successful.
+     */
+    static unsigned int token_count( NVStrings& strs, NVStrings& delimiter, unsigned int* results, bool devmem=true );
```

Comment:
It should be possible to declare both `NVStrings` parameters here and in the .cu file as `const`.
Also, recommend naming the `delimiter` parameter as `delimiters` to make it clear that more than one is possible.
```
static unsigned int token_count( NVStrings const& strs, NVStrings const& delimiters, unsigned int* results, bool devmem=true );
```
The doxygen comment should be updated to reflect this as well.